### ISSUE TITLE
Datatables pagination has accessibility issues (Course reserves page)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#981]
The pagination for the datatables is odd...
- [ ] each `<li>` has tabindex="0", so it gets keyboard focus but is not actionable, then the `<a>` within the `<li>` gets focus and is actionable. The tabindex should be removed. 
- [ ] list items that don't do anything should not be links:  Previous (when it's disabled); current page; "..."

The pagination is correct for most SW pages (except for the "..." which is still a link and shouldn't be). Don't know if it was missed for Course reserves, or if it's a different implementation altogether because of the datatable.

**Comment by jvine** [comment]
Fixed
